tests/runner/testrunner.go

```diff
@@ -52,31 +52,29 @@ func NewTestRunner(id string, apps []kube.AppDescription, comps []kube.Component
 }
```

Comment:
As I leave the comments. we would need to create the subdirectory and the separate packages. Then we do not need to change this test runner.